core/core.go

```diff
@@ -140,14 +125,25 @@ func RunBazeliskWithArgsFunc(argsFunc ArgsFunc, repos *Repositories) (int, error
 		if err != nil {
 			return -1, fmt.Errorf("could not get the list of incompatible flags: %v", err)
 		}
-
 		if args[0] == "--migrate" {
 			migrate(bazelPath, args[1:], newFlags)
-		} else {
+		} else if args[0] == "--migrate" {
```

Comment:
you mean --strict?

Reply:
The else alone is still fine, given the outer if

Reply:
Oh, right, or I should just remove this check.

Reply:
Done